docs/user/05-70-manage-kyma-environment-through-cis-provisioning-service.md

```diff
@@ -110,34 +114,35 @@ The SAP Cloud Management service (technical name: `cis`) provides the Provisioni
 
     kubectl should return the list of Pods in the `default` namespace running in the cluster, which means that the cluster is accessible.
 
-> [!NOTE]
-> The following steps are optional and show how to revoke the credentials by deleting the binding.
+## Next Steps
 
-12. List all bindings for the instance.
+> [!NOTE]
+> For extra security, you can perform the optional steps to delete the binding sooner than it is set to expire in the **EXPIRATION_SECONDS** environment variable.
+> Otherwise, the binding is automatically deleted after the maximum allowed expiration time (7200 seconds) passes.
 
-    ```bash
-    curl -s "$PROVISIONING_SERVICE_URL/provisioning/v1/environments/$INSTANCE_ID/bindings" -H "accept: application/json" -H "Authorization: bearer $TOKEN"
-    ```
+1.   (Optional) List all bindings for the instance.
 
-13. Delete the binding to revoke the credentials.
+      ```bash
+      curl -s "$PROVISIONING_SERVICE_URL/provisioning/v1/environments/$INSTANCE_ID/bindings" -H "accept: application/json" -H "Authorization: bearer $TOKEN"
+      ```
 
-    ```bash
-    curl -s -X DELETE "$PROVISIONING_SERVICE_URL/provisioning/v1/environments/$INSTANCE_ID/bindings/$BINDING_ID" -H "accept: application/json" -H "Authorization: bearer $TOKEN"
-    ```
+2.   (Optional) Delete the binding to revoke the credentials.
```

Comment:
Let's make it a first step in Next Steps.

Reply:
I'll move it up to the Steps and add the information that that's how you view the binding details.